cluster/delegate.go

```diff
@@ -24,9 +24,13 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-// Maximum number of messages to be held in the queue.
-const maxQueueSize = 4096
+const (
+	// Maximum number of messages to be held in the queue.
+	maxQueueSize = 4096
```

Comment:
Can you run gofmt on this?

```diff
diff -u ./cluster/delegate.go.orig ./cluster/delegate.go
--- ./cluster/delegate.go.orig	2019-01-25 18:08:08.853426105 +0000
+++ ./cluster/delegate.go	2019-01-25 18:08:08.853426105 +0000
@@ -27,10 +27,10 @@
 const (
 	// Maximum number of messages to be held in the queue.
 	maxQueueSize = 4096
-	fullState = "full_state"
-	update = "update"
-
+	fullState    = "full_state"
+	update       = "update"
 )
+
 // delegate implements memberlist.Delegate and memberlist.EventDelegate
 // and broadcasts its peer's state in the cluster.
 type delegate struct {
```

That should fix the CI failures.

Reply:
done